packages/react-compose-enhancers/README.md

```diff
@@ -0,0 +1,84 @@
+# `@shopify/react-compose-enhancers`
```

Comment:
I'm not sold on the package name. `compose` is nice, IMO, albeit generic. Maybe even `react-compose`?